### ISSUE TITLE
Metatag Type - Layout?

### ISSUE THREAD

**Issue by laryn** [backdrop-contrib/metatag#39]
I'm not completely sure that this is a good idea but thought I'd jot it down.

I was working on Metatags for a particular site and was thinking it would be helpful at times to be able to specify default metatags for a particular layout. I imagine there could be some duplication and conflicts (e.g. a default metatag for the "Global: Homepage", as well as for the layout that was the homepage layout...) but at times it could be helpful (such as layouts that define their own paths).

**Comment by jenlampton** [comment]
Yes, this is a good idea. We have a `metatag views`, and there was a `metatag panels` (in Drupal 7) that provide similar tags for other types of pages. I bet some of that code can be repurposed.

**Comment by herbdool** [comment]
Here's `metatag_panels` https://git.drupalcode.org/project/metatag/-/tree/7.x-2.x/metatag_panels?ref_type=heads
